backend/vkt/src/main/java/fi/oph/vkt/service/PublicEnrollmentService.java

```diff
@@ -70,11 +71,46 @@ private long getParticipants(final List<Enrollment> enrollments) {
       .count();
   }
 
+  @Transactional(readOnly = true)
+  public PublicEnrollmentInitialisationDTO getEnrollmentInitialisationDTO(long examEventId, Person person) {
+    final ExamEvent examEvent = examEventRepository.getReferenceById(examEventId);
+    final Optional<Reservation> reservationMaybe = reservationRepository.findByExamEventAndPerson(examEvent, person);
+    final Optional<Enrollment> enrollmentMaybe = findEnrollment(examEvent, person, enrollmentRepository);
```

Comment:
Toisaalla käytetty muotoa `maybeReservation` (maybePerson nyt ainakin). Vois käyttää täällä samaa. Itse oon tainnut nimetä näitä myös jossain `optionalEnrollment` tms. tavalla mut maybe ok